resources/monitoring/profile-production.yaml

```diff
@@ -0,0 +1,27 @@
+prometheus:
+  prometheusSpec:
+    resources:
+      limits:
+        cpu: 600m
+        memory: 1500Mi
+      requests:
+        cpu: 150m
+        memory: 1000Mi
+    retention: 30d
+    retentionSize: 15GB
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 20Gi
```

Comment:
Shouldn't we reduce the size heavily on evaluation profile?

Reply:
that is the production profile. This was copied from:
https://github.com/kyma-project/kyma/blob/19a922844fe8b30d8f251f33dabdebed1bbd75c1/installation/resources/installer-config-production.yaml.tpl#L67